package.json

```diff
@@ -60,6 +60,7 @@
   "resolutions": {
     "babel-core": "7.0.0-bridge.0",
     "domelementtype": "1.3.1",
+    "graceful-fs": "4.2.3",
```

Comment:
This is a fix for this [node 12](https://circleci.com/gh/artsy/force/38793) deploy error we are seeing. 

Found solution here: https://stackoverflow.com/a/58394828/1038901